pkg/resolution/common/annotations.go

```diff
@@ -22,4 +22,18 @@ const (
 	// AnnotationKeyContentType is the annotation key passed back
 	// with a resolved resource's content type.
 	AnnotationKeyContentType = resolution.GroupName + "/content-type"
+
+	// AnnotationKeyURL is the remote URL used
+	AnnotationKeyURL = resolution.GroupName + "/url"
+
+	// AnnotationKeyPath is the path used
+	AnnotationKeyPath = resolution.GroupName + "/path"
+
+	// AnnotationKeyRevision is the commit hash that was fetched
+	// from remote location
+	AnnotationKeyRevision = resolution.GroupName + "/revision"
+
+	// AnnotationKeyHashAlgo is the cryptographic hash algorithm being used
+	// to calculate the actual sha
+	AnnotationKeyHashAlgo = resolution.GroupName + "/hashAlgo"
```

Comment:
@wlynch RE: https://github.com/tektoncd/chains/pull/554#discussion_r974576447

I am trying to add new annotation here to pass the hash algo from resolver side. And for git source, I just created a constant with value `sha1` to set this annotation. Do you have any other better suggestion please? Thanks